cypress.config.ts

```diff
@@ -10,4 +10,8 @@ export default defineConfig({
   videosFolder: './cypress/videos',
   downloadsFolder: './cypress/downloads',
   fileServerFolder: './',
+  env: {
+    bash: 'C:\\Program Files\\Git\\bin\\bash.exe',
+    comSpec: 'C:\\Windows\\system32\\cmd.exe',
+  },
```

Comment:
required for the `cy.exec()` work around on windows machines

Reply:
This is only a Windows issue, so this bit of code checks the platform and if its not `win32` then run `cy.exec()` like normal. The bug comes from an issue on Cypress's side where they insert characters into windows paths leading to cypress not being able to find the command line bin.
https://github.com/timmyichen/dev-directory/blob/47e962d85acb025769c1662c74ba0cee5b4830c0/cypress/support/e2e.ts#L19-L23